sklearn/tests/test_isotonic.py

```diff
@@ -336,7 +336,7 @@ def test_isotonic_regression_oob_raise():
     ir.fit(x, y)
 
     # Check that an exception is thrown
-    msg = "A value in x_new is below the interpolation range"
+    msg = r"A value.* in x_new is below the interpolation range's minimum value .*\."
```

Comment:
```suggestion
    msg = "in x_new is below the interpolation range"
```

Reply:
Since the CI fails, I think that we make a partial match or feel free to fix the regex pattern.